# Can one block of except statements handle multiple exception ?

# Yes, You Can Use a Single  Block to Handle Multiple Exceptions in Python. 